sample_cgfs/rv32ip.cgf

```diff
@@ -2181,6 +2267,7 @@ zunpkd832:
 # pkbb16:
 #     config:
 #       - check ISA:=regex(.*I.*P.*)
+#       - check ISA:=regex(.*Zicsr.*); def pext_check_vxsat_ov=True
```

Comment:
Going with the changes in the `template.yaml` shouldn't the check directly check for the `Zicsr` in the previous line itself for all the instructions. The `pext_check_vxsat_ov` macro is no longer relevant right?

Reply:
Enabling the macro based on `Zicsr` doesn't make sense. Because `Zicsr` is a required extension for `P`. We might need an extra field somewhere in the input isa yaml which indicates whether the optional saturation flag is implemented or not. The macro will have to be enabled based on that field and not `Zicsr`.

Reply:
> Enabling the macro based on `Zicsr` doesn't make sense. Because `Zicsr` is a required extension for `P`. We might need an extra field somewhere in the input isa yaml which indicates whether the optional saturation flag is implemented or not. The macro will have to be enabled based on that field and not `Zicsr`. I agree that there needs to be a guard which prevents spurious `mstatus` accesses.

Okay, I removed the pext_check_vxsat_ov check. For the saturation flag, it is not optional in `P`: the spec just does not require mstatus.VS to be implemented, but requires it to guard against vxsat accesses if implemented. Since VS is WARL, there is no need to have extra checks against mstatus.